sites/docs/src/content/docs/usage/configuration/introduction.md

```diff
@@ -0,0 +1,178 @@
+
+### Shared configuration files
+
+An `includeConfig` statement in the `nextflow.config` file is also used to include custom institutional profiles that have been submitted to the [nf-core config repository](https://github.com/nf-core/configs). At run time, nf-core pipelines will fetch these configuration profiles from the [nf-core config repository](https://github.com/nf-core/configs) and make them available.
+
+For shared resources such as an HPC cluster, you may consider developing a shared institutional profile.
+
+Follow [this tutorial](https://nf-co.re/docs/usage/tutorials/step_by_step_institutional_profile) to set up your own institutional profile.
+
+### Custom parameter and configuration files
+
+Nextflow will look for files that are external to the pipeline project directory. These files include:
+
+- The config file `$HOME/.nextflow/config`
+- A config file named `nextflow.config` in your current directory
+- Custom configuration files specified using the command line
+  - A parameter file that is provided using the `-params-file` option
+  - A config file that are provided using the `-c` option
+
+**Parameter file format**
+
+Parameter files are `.json` files that can contain an unlimited number of parameters:
+
+```json title="nf-params.json"
+{
+  "<parameter1_name>": 1,
+  "<parameter2_name>": "<string>",
+  "<parameter3_name>": true
+}
+```
+
+You can override default parameters by creating a `.json` file and passing it as a command-line argument using the `-param-file` option.
+
+```bash
+nextflow run nf-core/rnaseq -profile docker --input <path/to/input? --outdir <results> -param-file <path/to/nf-params.json>
+```
+
+**Configuration file format**
+
+Configuration files are `.config` files that can contain various pipeline properties and can be passed to Nextflow using the `-c` option in your execution command:
+
+```bash
+nextflow run nf-core/rnaseq  -profile docker --input <path/to/input> --outdir <results> -c <path/to/custom.config>
+```
+
+Custom configuration files are the same format as the configuration file included in the pipeline directory.
+
+Configuration properties are organized into scopes by dot prefixing the property names with a scope identifier or grouping the properties in the same scope using the curly brackets notation. For example:
+
+```groovy
+alpha.x  = 1
+alpha.y  = 'string value'
+```
+
+Is equivalent to:
+
+```groovy
+alpha {
+    x = 1
+    y = 'string value'
+}
+```
+
+### Scopes
+
+Scopes allow you to quickly configure settings required to deploy a pipeline on different infrastructure using different software management.
+
+A common scenario is for users to write a custom configuration file specific to running a pipeline on their infrastructure.
+
+:::warning
```

Comment:
Ah this should go above, but I have a feeling talking ab out `scopes` here might be getting into too much detail, maybe drop this whole section to say 'to see all options for how to configure your pipeline see the nextflow docs'

Reply:
I think it needs to be included, and I agree it is potentially too much detail, but I think it's important to frame that they exist, can be added to the same config file, and address different configuration needs.

I try to stay away from params here to prevent confusion.

Reply:
I've merged this into the config file descritpion to try to improve the flow.